sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/__init__.py

```diff
@@ -32,6 +32,12 @@
     RecognizePiiEntitiesResult,
     PiiEntity,
     PiiEntityDomainType,
+    RecognizeHealthcareEntitiesResult,
+    EntitiesRecognitionTask,
+    PiiEntitiesRecognitionTask,
+    EntityLinkingTask,
+    KeyPhraseExtractionTask,
+    SentimentAnalysisTask
```

Comment:
you also need to include these models in the __all__

Reply:
Added!